aws/ecs-service/iam.tf

```diff
@@ -0,0 +1,47 @@
+/* 
+ *  IAM role for fargate instances to use when they are launched
+ */
+
+
+data "aws_iam_policy_document" "ecs_tasks_execution_role_policy" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["ecs-tasks.amazonaws.com"]
+    }
+  }
+}
+
+
+resource "aws_iam_role" "ecs_tasks_execution_role" {
+  name               = "TradeTariff-${var.environment}-ecs-task-execution-role"
```

Comment:
We'll run into naming clashes here as multiple tasks use the module, should we have this name generated using more input variables?